packages/fxa-auth-server/test/local/routes/account.js

```diff
@@ -3814,6 +3814,25 @@ describe('/account/destroy', () => {
     });
   });
 
+  it('should delete the password less account', () => {
+    mockDB = { ...mocks.mockDB({ email: email, uid: uid, verifierSetAt: 0 }) };
```

Comment:
nit: Not sure if we have a style guide rule around shorthand syntax, but 🤷&hellip;

```suggestion
    mockDB = { ...mocks.mockDB({ email, uid, verifierSetAt: 0 }) };
```